Mathlib/Topology/ShrinkingLemma.lean

```diff
@@ -47,7 +47,7 @@ This type is equipped with the following partial order: `v ≤ v'` if `v.carrier
 and `v i = v' i` for `i ∈ v.carrier`. We will use Zorn's lemma to prove that this type has
 a maximal element, then show that the maximal element must have `carrier = univ`. -/
 -- Porting note(#5171): this linter isn't ported yet. @[nolint has_nonempty_instance]
-@[ext] structure PartialRefinement (u : ι → Set X) (s : Set X) where
+@[ext] structure PartialRefinement (u : ι → Set X) (s : Set X) (p : Set X → Prop) where
```

Comment:
Do you think you could split the refactor of adding this `p` argument to a separate PR?

Reply:
Thanks for your review! Sure, I will do it

Reply:
#18827